src/CMakeLists.txt

```diff
@@ -286,6 +286,9 @@ if(EXIV2_ENABLE_INIH)
   list(APPEND requires_private_list "INIReader")
 endif()
 
+target_link_libraries(exiv2lib_int PUBLIC fmt::fmt)
+target_link_libraries(exiv2lib PUBLIC fmt::fmt)
```

Comment:
Gate by C++20?

Should be PRIVATE?

Also need to add  to .pc file Requires.private or Libs.private.

Reply:
Done